sdk/python/tests/facade/test_BatchOperations.py

```diff
@@ -54,19 +56,22 @@ def test_can_load_all_transactions(self):
 		# Assert:
 		self.assertEqual(2, len(transactions))
 
-		self.assertEqual(0x0101, transactions[0].type)
-		self.assertEqual(0x98000001, transactions[0].version)
-		self.assertEqual(TEST_PUBLIC_KEY, transactions[0].signer_public_key)
-		self.assertEqual(ALICE_ADDRESS, transactions[0].recipient_address)
-		self.assertEqual(3000000, transactions[0].amount)
-		self.assertEqual(b'Hello world!', transactions[0].message)
+		self.assertEqual(0x0101, transactions[0].type_.value)
+		self.assertEqual(2, transactions[0].version)
+		self.assertEqual(0x98, transactions[0].network.value)
 
-		self.assertEqual(0x0101, transactions[1].type)
-		self.assertEqual(0x98000001, transactions[1].version)
-		self.assertEqual(TEST_PUBLIC_KEY, transactions[1].signer_public_key)
-		self.assertEqual(BOB_ADDRESS, transactions[1].recipient_address)
-		self.assertEqual(1000000, transactions[1].amount)
-		self.assertEqual(None, transactions[1].message)
+		self.assertEqual(TEST_PUBLIC_KEY.bytes, transactions[0].signer_public_key.bytes)
+		self.assertEqual(str(ALICE_ADDRESS).encode('utf8'), transactions[0].recipient_address.bytes)
```

Comment:
are these different types ?

Reply:
y, `ALICE_ADDRESS` is sdk Address
other one is catbuffer's (nc.)Address,

moreover later requires 40b 😅